src/components/SQFormGuidedWorkflow/SQFormGuidedWorkflow.js

```diff
@@ -18,6 +18,9 @@ import {GuidedWorkflowProps} from './PropTypes';
 
 const useStyles = makeStyles(() => {
   return {
+    root: {
+      padding: '16px !important' /* material-ui override */
```

Comment:
The `!important` tag was needed in version 0.x of Material-UI but shouldn't be needed when using `makeStyles`
Can you try this without the `!important` tag please?

Reply:
let me try

Reply:
I removed !important but not working I tried.